$NetBSD: patch-config_stompserver.conf,v 1.2 2012/08/13 18:43:34 imil Exp $

Decent configuration file with user privileges separation and proper
filesystem locations

--- config/stompserver.conf.orig	2011-03-22 12:52:14.000000000 +0000
+++ config/stompserver.conf
@@ -1,11 +1,14 @@
 ---
-:daemon: false
-:working_dir: /tmp/stompserver
-:storage: .queue
-:queue: file
+:daemon: true
+:working_dir: @STOMPSERVER_WRKDIR@
+:logdir: @STOMPSERVER_LOGDIR@
+:pidfile: @STOMPSERVER_PIDFILE@
+:etcdir: @STOMPSERVER_CFGDIR@
+:storage: @STOMPSERVER_STORAGE@
+:queue: memory
 :auth: false
 :debug: false
-:group:
-:user:
+:group: @STOMPSERVER_GROUP@
+:user: @STOMPSERVER_USER@
 :host: 127.0.0.1
 :port: 61613
